CHANGES.md

```diff
@@ -1,3 +1,11 @@
+0.12.0 (2024-12-11)
+-------------------
+- We now prefer using calibrations, specifically arcs, from the same block or the same proposal
+- We only use public calibrations now
```

Comment:
These two statements seem in conflict unless they have differing meanings for the word "now". 
Can you clarify a little?
If a proposal isn't public, then we wouldn't be able to use calibrations from the same block...